.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ on:
     push:
         tags:
             - "*"
-
+    workflow_dispatch:
```

Comment:
dont we need to give `inputs` here?

Reply:
Not for this one. This will let us trigger releases in the event that one fails